src/cmd/create.js

```diff
@@ -7,25 +7,25 @@ const CWD = process.cwd()
 const { getPropertyId } = require('../lib/get-resource-ids')
 const { getPublishedTemplates } = require('../lib/get-templates')
 
-module.exports = async function create (pid) {
+module.exports = async function create (pid, xpName, traffic, selectedTemplate = null) {
```

Comment:
can we change xpName to `name` and selectedTemplate to `templateId` for consistency with the rest of the codebase?